plugins/code-versions/src/styles.css

```diff
@@ -0,0 +1,34 @@
+@import "framer-plugin/framer.css";
```

Comment:
Hmm, I think this won't work well, because unlayered rules take precedence over layered ones, which is all of Tailwind. Take a look at `globals.css` in HubSpot, for inspiration.

Reply:
Exactly that's why, yes. I considering dropping this one completely, but decided it's better to have some defaults in, while maybe some rules need to be overwritten gloabaly.